copy/copy.go

```diff
@@ -1064,6 +1145,52 @@ func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, to
 		}
 	}
 
+	// First look for a delta that matches this layer and substitute the result of that
+	if ok, deltaResultMediaType := ic.canUseDeltas(srcInfo); ok {
```

Comment:
Interaction for encrypting path with `toEncrypt` flag

Reply:
Not sure what you mean here, it seem alright to me. Applying deltas naturally create the uncompressed layers, so we get the diffid etc. That said, I'm not well versed in how crypto works here, so I might be missing something.

Reply:
Hmm... so the `toEncrypt` flag tells the copy process to encrypt the blob when passed into `copyBlobFromStream`. My concern was if any deltas are created by this process, that they would need to support encryption. If they are not, then it should probably be fine.

Reply:
copy never generates deltas, it just applies them to regenerate the tar. Creating deltas is currently done by a separate patch to skopeo.